modules/kernel/src/org/apache/axis2/transport/http/MultipartFormDataFormatter.java

```diff
@@ -258,6 +263,24 @@ public String formatSOAPAction(MessageContext messageContext, OMOutputFormat for
                     part = new FilePart(fieldName,
                             new ByteArrayPartSource(filename, Base64.decodeBase64(ele.getText().getBytes())),
                             contentType, charset);
+                } else if ((ele.getAttribute(FILENAME_ATTRIBUTE_QNAME) != null)) {
```

Comment:
This is equal to the next else if block. Let's only introduce the use of the parameter 'DECODE_MULTIPART_DATA_PARAM'